__tests__/app.test.js

```diff
@@ -47,4 +47,39 @@ describe('GET /api',() => {
             expect(body).toEqual(endPoints)
         })
     })
-})
\ No newline at end of file
+})
+describe('GET /api/articles/:article_id',() => {
+    test('status 200, Responds with an article object with properties of requested id', () => {
+        return request(app)
+        .get('/api/articles/1')
+        .expect(200)
+        .then(({body}) => {
+            const article  = body;
```

Comment:
We could potentially be a bit more specific here. Rather than just checking for the keys we could ensure the response has values attached to them as well. Your test is 'okay' as is but if you wanted to you could make it sturdier. If you want you could hard code the exact response we expect as we have the data in the test file, or at least expect.any(String) for example for each value (except for the id, as we should be asserting that we expect this to be 1)